templates/front/base_docs.html

```diff
@@ -22,6 +32,9 @@
         <li {% if section == "api" %} class="active" {% endif %}>
             <a href="{% url 'hc-docs-api' %}">REST API</a>
         </li>
+        <li {% if section == "faq" %} class="active" {% endif %}>
+            <a href="{% url 'hc-docs-faq' %}">FAQ</a>
```

Comment:
Be consistent with the quotation marks you use. choose between double or single.

Reply:
Hey @SerryJohns , I beg to differ, in this particular `line 36`, the two types of quotes are used appropriately. The `'hc-docs-faq'` are nested under the `href=""` property of the `<a>,<\a>` tag.

Reply:
cool